frontend/src/container/Sidebar.jsx

```diff
@@ -37,23 +43,20 @@ export class Sidebar extends React.Component {
     this.state = {
       activeWorkspaceId: NO_ACTIVE_SPACE_ID,
       foldedSpaceList: [],
-      sidebarClose: isMobile
+      isSidebarClose: isMobile,
+      showSpaceList: true,
+      showUserItems: true
     }
 
     props.registerCustomEventHandlerList([
-      { name: CUSTOM_EVENT.SHOW_CREATE_WORKSPACE_POPUP, handler: this.handleShowCreateWorkspacePopup }
+      { name: CUSTOM_EVENT.SHOW_CREATE_WORKSPACE_POPUP, handler: this.handleClickNewWorkspace }
```

Comment:
```suggestion
      { name: CUSTOM_EVENT.SHOW_CREATE_WORKSPACE_POPUP, handler: this.handleClickNewSpace }
```
I'd suggest to rename the handleClickNewWorkspace function to handleClickNewSpace